src/main/java/cc/hyperium/gui/playerrenderer/FakePlayerRendering.java

```diff
@@ -42,46 +39,10 @@ public FakePlayerRendering(GameProfile profile) {
         player = new FakePlayer(profile);
     }
 
-    public void renderOtherPlayerModel(AbstractClientPlayer givenPlayer, int posX, int posY, float scale, float rotation) {
-        Minecraft mc = getMinecraft();
```

Comment:
why are you messing with this

Reply:
its not used anywhere
nothing is breaking because of it

Reply:
👍